main/libressl/APKBUILD

```diff
@@ -2,9 +2,9 @@
 # Contributor: Orion <systmkor@gmail.com>
 # Maintainer: Orion <systmkor@gmail.com>
 pkgname=libressl
-pkgver=2.5.3
+pkgver=2.5.4
 _namever=${pkgname}${pkgver%.*}
-pkgrel=0
+pkgrel=1
```

Comment:
If you're bumping the version, the pkgrel should be reset to 0 -- but that should be a separate commit if you do, possibly a separate PR, since I recall there being at least one problematic change in LibreSSL 2.5.4 w.r.t. `getrandom()` usage. I'd recommend leaving it at 2.5.3 for this PR.